src/Model/Table/SoftDeleteTrait.php

```diff
@@ -7,6 +7,20 @@
 trait SoftDeleteTrait {
 
     /**
+     * Get the configured deletion field
+     *
+     * @return string
+     */
+    protected function getField()
```

Comment:
Isn't `getField` a too common name for a Table method? If a table using softDeleteTrait already has a `getField()` method, soft deleting would then fail.
I would use a less common name, like `getSoftDeleteField()`. What do you think?